bciers/apps/administration/app/data/jsonSchema/contact.ts

```diff
@@ -1,7 +1,7 @@
 import { RJSFSchema } from "@rjsf/utils";
 import provinceOptions from "@bciers/data/provinces.json";
 import SectionFieldTemplate from "@bciers/components/form/fields/SectionFieldTemplate";
-import { InlineArrayFieldTemplate } from "@bciers/components/form/fields";
+import { ArrayFieldTemplate } from "@bciers/components/form/fields";
```

Comment:
Why is it better to use this one?

Reply:
I'm not entirely sure why it was originally used, but these two field templates are quite similar. When I used them interchangeably, they appeared identical. The only noticeable difference was the note prop, which I assume we would also need in the ArrayField. That said, I aimed to minimize the usage of InlineArrayFieldTemplate and instead reuse ArrayFieldTemplate.

Additionally, the removable prop on the ArrayFieldTemplate perfectly aligned with the requirements of this feature. I avoided modifying InlineArrayFieldTemplate to replicate the same removable prop behaviour, as it seemed unnecessary.